docs/getting_started/first_example.md

```diff
@@ -6,8 +6,9 @@ familiarize yourself with the SDK and the Matter ecosystem.
 ## Example Devices
 
 The example devices (occasionally referred to as "apps") are located in the
-[examples](../../examples/) directory. The examples often implement one
-particular device type. Some have implementations for various platforms.
+[examples](https://github.com/project-chip/connectedhomeip/tree/master/examples)
```

Comment:
Let me preface that I am not an expert in `.md` files at all

Not sure I am following this fix to the examples link. When I click the link on examples over in the web rendered version of this md file over here https://github.com/project-chip/connectedhomeip/blob/master/docs/getting_started/first_example.md I am getting the examples directory. It seems like the link is working and I would think relative path is better in case of a local copy of the repo

Reply:
That leads to a 404. Okay I see what the issue is now. Thanks for that

Reply:
It seems like a weird bug that only happens when you access that link from  https://project-chip.github.io/connectedhomeip-doc/getting_started/first_example.html#an-sdk-example but not from https://github.com/project-chip/connectedhomeip/blob/master/docs/getting_started/first_example.md .

The weirdest part is that the link to `../../examples/chip-tool` works fine on both. But `../../examples/` breaks somehow.

Reply:
Unresolving this to see if we can figure this out

https://project-chip.github.io/connectedhomeip-doc/_sources/getting_started/first_example.md

That show that the URL is `(https://github.com/project-chip/connectedhomeip/blob/master/../examples)` for some reason

Reply:
Spent sometime trying to figure this out, but I honestly couldn't understand what the issue is. I am okay with leaving a comment about what we discovered about this like and just using and absolute path :shrug:

Reply:
I opened https://github.com/project-chip/connectedhomeip/issues/35855 and linked the discussion here so we don't lose the context.

Reply:
I'll pass that on to the folks doing the SDK / handbook crossover stuff. Surely someone knows how the magic happens

Reply:
Probably something in sphynx configuration: https://github.com/project-chip/connectedhomeip/blob/master/docs/conf.py#L76

I think I tried to figure it out before and it was never super clear...

Reply:
I added a comment to the code pointing to the separate bug I created.
I propose we land this PR as it fixes this link and other issues and use the specific bug to continue the discussion about the relative link issue.